lib/galaxy/objectstore/cloud.py

```diff
@@ -636,7 +641,7 @@ def _get_filename(self, obj, **kwargs):
         #         os.makedirs(cache_path)
         #     return cache_path
         # Check if the file exists in the cache first
```

Comment:
use same comment for consistency? (all this duplication is ready for refactoring, of course; but until then, to prevent confusion; unless there's a subtle difference I missed)
```suggestion
        # Check if the file exists in the cache first, always pull if file size in cache is zero
```